app/upgrades/v3/forks.go

```diff
@@ -22,9 +21,10 @@ func RunForkLogic(ctx sdk.Context, keepers *keepers.AppKeepers) {
 // Fixes an error where minimum deposit was set to "500 osmo". This denom does
 // not exist, which makes it impossible for a proposal to go to a vote.
 func FixMinDepositDenom(ctx sdk.Context, gov *govkeeper.Keeper) {
-	params := gov.GetDepositParams(ctx)
-	params.MinDeposit = sdk.NewCoins(sdk.NewCoin("uosmo", osmomath.NewInt(500000000)))
-	gov.SetDepositParams(ctx, params)
+	// UNFORKINGTODO OQ: No longer exists, how do we want to go about this?
+	// params := gov.GetDepositParams(ctx)
+	// params.MinDeposit = sdk.NewCoins(sdk.NewCoin("uosmo", osmomath.NewInt(500000000)))
+	// gov.SetDepositParams(ctx, params)
```

Comment:
This can just be left commented out forever -- its fork code.

Reply:
Changed it from UNFORKINGTODO to UNFORKINGNOTE and keeping commented out for posterity.

https://github.com/osmosis-labs/osmosis/pull/6413/commits/a21a8326013b855328b94cc99eaa099c2af135cf